src/rgw/rgw_op.cc

```diff
@@ -1404,16 +1467,31 @@ void RGWGetObj::execute()
 
   start = ofs;
 
+  if (need_decompress) {
+
+    if (cs_info.blocks.size() == 0) {
+      lderr(s->cct) << "ERROR: no info about compression blocks, cannot decompress" << dendl;
+      op_ret = -EIO;
```

Comment:
You see s3tests.functional.test_s3.test_object_head_zero_bytes and s3tests.functional.test_s3.test_object_copy_zero_size fail 
OR you get some error from lone 1473 ?

Because I dont see either one.